vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManager.java

```diff
@@ -253,17 +280,21 @@ public IChoiceBlock getChoiceBlock(String choiceBlockId) {
     /*
      * (non-Javadoc)
      * 
-     * @see edu.asu.diging.vspace.core.services.impl.IChoiceBlock#createTextBlock(java.
+     * @see
+     * edu.asu.diging.vspace.core.services.impl.IChoiceBlock#createTextBlock(java.
      * lang.String, java.lang.String, java.lang.Integer)
      */
     @Override
-    public IChoiceBlock createChoiceBlock(String slideId, List<String> selectedChoices, Integer contentOrder, boolean showsAll) {
+    public IChoiceBlock createChoiceBlock(String slideId, List<String> selectedChoices, Integer contentOrder,
+            boolean showsAll) {
         List<IChoice> choices = new ArrayList<IChoice>();
-        if(!showsAll) {
-            choices = selectedChoices.stream().map(choice -> slideManager.getChoice(choice)).collect(Collectors.toList());
+        if (!showsAll) {
+            choices = selectedChoices.stream().map(choice -> slideManager.getChoice(choice))
+                    .collect(Collectors.toList());
         }
-        IChoiceBlock choiceBlock = choiceBlockFactory.createChoiceBlock(slideManager.getSlide(slideId), contentOrder, choices, showsAll);
-        return choiceBlockRepo.save((ChoiceBlock)choiceBlock);
+        IChoiceBlock choiceBlock = choiceBlockFactory.createChoiceBlock(slideManager.getSlide(slideId), contentOrder,
+                choices, showsAll);
+        return choiceBlockRepo.save((ChoiceBlock) choiceBlock);
```

Comment:
has this method actually changed? if not, it shouldn't show up as changed.

Reply:
This method hasn't been changed. Not sure why this was showing as changed. I had revert back the code and replaced with the code in develop branch  for this class and just  appended my changes. So now it will not be an issue